R/traits/clean_trait_data.R

```diff
@@ -17,90 +17,319 @@ get_file(node = "pk4bg",
+# Add elevation
+# Fix experiment column when obviously wrong
+# Clean leaf thickness data which is wrong
+
+clean_traits2 <- clean_traits %>%
+  mutate(day = case_when( siteID == "Ulv" & project == "Incline" ~ 24,
+                          siteID == "Hog" & project == "3D" ~ 24,
+                          siteID == "Vik" & project == "3D" ~ 26,
+                          siteID == "Gud" & project == "Incline" ~ 27,
+                          siteID == "Lia" & project == "3D" ~ 28,
+                          siteID == "Skj" & project == "Incline" ~ 30,
+                          siteID == "Joa" & project == "3D" ~ 1,
+                          TRUE ~ day),
+         siteID = case_when(day == 24 & project == "Incline" ~ "Ulv",
+                            day == 24 & project == "3D" ~ "Hog",
+                            day == 26 & project == "3D" ~ "Vik",
+                            day == 27 & project == "Incline" ~ "Gud",
+                            day == 28 & project == "3D" ~ "Lia",
+                            day == 30 & project == "Incline" ~ "Skj",
+                            day == 1 & project == "3D" ~ "Joa",
+                            TRUE ~ siteID),
+         project = case_when(day == 24 & siteID == "Ulv" ~ "Incline",
+                             day == 24 & siteID == "Hog" ~ "3D",
+                             day == 26 & siteID == "Vik" ~ "3D",
+                             day == 27 & siteID == "Gud" ~ "Incline",
+                             day == 28 & siteID == "Lia" ~ "3D",
+                             day == 30 & siteID == "Skj" ~ "Incline",
+                             day == 1 & siteID == "Joa" ~ "3D",
+                             TRUE ~ project),
+         year = 2022,
+         month = if_else(day == 1, 8, 7),
+         date = make_date(year, month, day),
+         elevation_m_asl = case_when(siteID == "Ulv" ~  1208,
+                                     siteID == "Hog" ~ 700,
+                                     siteID == "Vik" ~ 469,
+                                     siteID == "Gud" ~ 1213,
+                                     siteID == "Lia" ~ 1290,
+                                     siteID == "Skj" ~ 1088,
+                                     siteID == "Joa" ~ 920),
+         experiment = ifelse(experiment == "NA", NA, experiment),
+         leaf_thickness_1_mm = if_else(ID == "IKY0250", 0.207, leaf_thickness_1_mm),
+         leaf_thickness_1_mm = if_else(ID == "DEV8302", 0.155, leaf_thickness_1_mm),
+         leaf_thickness_2_mm = if_else(ID == "CZW4480", "0.153", leaf_thickness_2_mm),
+         leaf_thickness_2_mm = if_else(ID == "DDI9716", "0.223", leaf_thickness_2_mm),
+         leaf_thickness_2_mm = if_else(ID == "DEX5838", "0.185", leaf_thickness_2_mm),
+         leaf_thickness_3_mm = if_else(ID == "CHV2350", 0.198, leaf_thickness_3_mm),
+         leaf_thickness_2_mm = as.numeric(leaf_thickness_2_mm),
+         plant_height = ifelse(project == "Incline", plant_height/10,plant_height),
```

Comment:
Can you please add a comment here saying why this is done. All incline plants were measured in mm, so convert to cm